gateway/src/args.rs

```diff
@@ -23,6 +26,72 @@ pub(crate) trait Args {
     fn hot_reload(&self) -> bool;
 
     fn log_style(&self) -> LogStyle;
+
+    fn graph_ref(&self) -> Option<&GraphRef>;
+
+    fn dashboard_analytics_enabled(&self) -> bool {
```

Comment:
would rename it to `can_export_analytics_to_platform()` or something like this. This sounds like a feature the user chooses to use or not.

Reply:
I would use the word telemetry rather than analytics